### DISCUSSION TITLE
Problem with deformed FFD visulaization

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#325] [Bug]
Hi,

I add the command "writeDeformedFFDs: True" to the runscript at the same line like version 2, but the following error occurs.

I have attached the runscript and the log file for the error. When I delete command, the case runs successfully.

Thank you.

[logOpt.txt](https://github.com/mdolab/dafoam/files/9255619/logOpt.txt)
[runScript.zip](https://github.com/mdolab/dafoam/files/9255622/runScript.zip)

**Comment by friedenhe** [comment]
Please upload the entire optimization folder such that we can run and reproduce the issue you had. You can use a toy problem too. We can't run the case with only runScript.py

**Reply by hosseinnegahban** [reply]
[test.zip](https://github.com/mdolab/dafoam/files/9255651/test.zip)

**Reply by hosseinnegahban** [reply]
Please find here the whole optimization file

**Reply by friedenhe** [reply]
OK, this is a bug in v3.0.0. The deformedFFD option is not working. The temporary workaround is to use `prob.model.geometry.DVGeo.writeTecplot("deformedFFD.dat")` after `prob.run_driver()` in runScript.py. This will output the FFD for the optimized design. We will fix this bug in a future version.

**Comment by hosseinnegahban** [comment]
Thank you very much for your reply. Then I will use the alternative way you mentioned.
________________________________
